### ISSUE TITLE
Known good .desktop file doesn't launch from a pie slice

### ISSUE THREAD

**Issue by Bobuntuu** [Schneegans/Gnome-Pie#190]
Ubuntu 20.04 with Unity desktop.

I installed Gnome Pie as a quick route to a number of free VPN servers at ProtonVPN.
I wrote a .desktop file with the command to run the Proton CLI utility, 'protonvpn-cli connect', which works fine from Unity launcher.
I also wrote new .desktop files for nine of their free servers in the Netherlands: 'protonvpn-cli c NL-FREE#[1-9]', which also work fine from Unity launcher. One more file to disconnect (protonvpn-cli d) and the set was complete.
With everything tested in Unity, I populated a new pie with all of the above launchers (11) and they all work perfectly, except the one to launch the cli utility. As I said above, it runs fine from Unity launcher, but nothing happens when selected from a pie.

**Comment by Schneegans** [comment]
Can you try editing the faulty item (just click on it) and check whether the correct command was loaded from the desktop file? You may also  starting Gnome-Pie from the command line (`killall gnome-pie && gnome-pie`) and select the item. Maybe an error is printed?

**Comment by Bobuntuu** [comment]
Thanks for the quick response.
Running, 'gnome-pie' from terminal, the link/slice works fine and the cli opens as it should. Opening Gnome-pie from the app menu or auto-start, I get no response from that slice of the pie. However, if there  is already a connection to a Proton VPN server, selecting the link will disconnect me, as it should, but the cli doesn't open. What's confusing me is that all the other slices work perfectly with direct server access and the 'disconnect' command. I see from the 'exec' entry of gnome-pie.desktop that the command (gnome-pi) is the same.
I don't know if this will help...

[Desktop Entry]
Name=ProtonVPN
Comment=Open ProtonVPN cli.
GenericName=ProtonVPN
Exec=protonvpn-cli connect
Terminal=true
Type=Application
Categories=Internet
Icon=/home/z/gnome_furniture/ProtonVPN.jpg
OnlyShowIn=Unity;
X-AppStream-Ignore=true
X-Ubuntu-Gettext-Domain=session-shortcuts

**Comment by Bobuntuu** [comment]
Sorry, you asked me to, "check whether the correct command was loaded from the desktop file".
Yes, I had checked that, but I had to restart the app after every edit of the .desktop file for the command to update in the slice.
The only difference with this command, compared to those that work, is that this is the only one which should open the terminal. As I said previously, the VPN does get disconnected, so it's running the command, but the terminal doesn't open.
